portal-web/test/functional/com/liferay/portalweb/macros/FormRules.macro

```diff
@@ -296,6 +296,20 @@ definition {
 			locator1 = "FormRules#RULE_BUILDER_BROKEN");
 	}
 
+	macro viewBuilderRuleAction {
+		AssertElementPresent(
+			key_positionNumber = "${positionNumber}",
+			key_value = "${value}",
+			locator1 = "FormRules#RULE_BUILDER_FIELD_GENERIC");
+	}
+
+	macro viewBuilderRuleCondition {
```

Comment:
@Diogomes, please rename to `viewRuleBuilderCondition` to keep consistency with the name used in the UI and in the path `RULE_BUILDER_FIELD_GENERIC`